sci-libs/gdal/gdal-2.2.4.recipe

```diff
@@ -28,9 +28,9 @@ COPYRIGHT="2007-2014 Even Rouault
 	2009-2014 Martin Landa
 	2013  ZJU Walkinfo Technology Corp., Ltd."
 LICENSE="MIT"
```

Comment:
Here is [the complete list][0] of all copyright holders and licenses from Debian. Please update the port accordingly

[0]: http://metadata.ftp-master.debian.org/changelogs/main/g/gdal/gdal_2.2.4+dfsg-1_copyright